### ISSUE TITLE
The Great Improvement List of Amr

### ISSUE THREAD

**Issue by lorenzolewis** [tauri-apps/typedoc-plugin-markdown#1]
CC @amrbashir 

- [x] remove the bread crumbs
- [x] defined in should be less important so maybe instead of a heading followd by a line it couldbe `**Defined in:** [cli.ts:24](url)`
- [x] ``` `example` ``` to `### Example:` for `@example` tag
- [x] Return is reduced from 3 lines to one 
- [x] Interface & Class rendering
- [x] Add option to disable rendering of the horizontal divider
- [x] don't document private fucntion/methods/properties
- [x] functions/constructors/methods/properties signature should be in a quote eg `> occurences: number`
- [x] functions/classes/enums/interfaces/TypeAliases header should be in a inline-code block

**Comment by lorenzolewis** [comment]
I personally prefer having the parameters as a table rather than a list, it just seems a bit easier to scan through and find the specific one you want (plus the structure is similar if there are multiple parameters)

**Comment by amrbashir** [comment]
That's fine by me. _I removed it from the list_

**Comment by amrbashir** [comment]
Since we changed function names to be in inline code block, we also need to change the style of `@example` tag from ``` `example` ``` to `### Example:` like in the second screenshot.  _I added it to the list_

**Comment by lorenzolewis** [comment]
Yeah, the example thing is on my list, it bothered me a bit. I had to work on it in the original repo a bit before so shouldn't be too bad.

**Comment by lorenzolewis** [comment]
I'm going to work on embedding interfaces, enums, and functions onto the module page and I _think_ # 2 in the first list will be resolved with that

**Comment by lorenzolewis** [comment]
Interfaces and Classes could use some TLC on rendering
<img width="648" alt="Screenshot 2022-09-11 at 14 53 57" src="https://user-images.githubusercontent.com/15347255/189531248-76edef3f-e736-441f-bdaa-6ad4d2dcb82f.png">
<img width="849" alt="Screenshot 2022-09-11 at 14 54 52" src="https://user-images.githubusercontent.com/15347255/189531297-977eef8b-0b31-441b-aa21-8e08e4962a95.png">
